### ISSUE TITLE
Feature suggestion: Add a route to create a comment on a pad

### ISSUE THREAD

**Issue by lpagliari** [JohnMcLear/ep_comments#26]
This is a feature suggestion, and I'll work on it. I just wanted to get some feedback before implementing it to make sure I'll address everything we need.

My suggestion is to create a route on the API to post a comment to a pad. So basically the steps would be:

* register a POST route `/p/:pad/:rev?/add/comment/` (or should it be `/p/:pad/:rev?/addComment/`?)
* from there we call `comments.addPadComment()` -- similar to [the code that is commented out on `ep_comments/index.js`](https://github.com/JohnMcLear/ep_comments/blob/f72b1b97705af02e74ab50df2fbdd60bf6f5e494/index.js#L102)
  * if err, should we return codes like [the ones used on the Etherpad API](http://etherpad.org/doc/v1.5.6/#index_response_format)? Something like `{ "code": 2, "message": "..." }`?
  * if success, we return `{ "code": 0, "commentId": "c-12345" }`
* finally, we need to tell the pad that the comment was created, so we broadcast 'pushAddComment' to the padId -- like what is done [here](https://github.com/JohnMcLear/ep_comments/blob/f72b1b97705af02e74ab50df2fbdd60bf6f5e494/index.js#L35). (I'm not sure how to do this step, but when I get to this point I'll see if I need some support here)

Is that it? Or do I need to do something else?

**Comment by JohnMcLear** [comment]
Sounds good.  Just remember to include some testing if you can! Thanks! J
----- Reply message -----

**Comment by lpagliari** [comment]
Ok, so I got stuck here:

* I've create a simple GET route, worked fine
* When I changed that to POST, I started getting `405: Only the HEAD or GET methods are allowed.`
* I've tried installing `ep_post_data` to make sure is not a problem with Nginx or my Etherpad version, worked fine too (pads were created as expected)

So after comparing `ep_comments` and `ep_post_data` for a while, I figured out that the problem was with `"pre": ["ep_etherpad-lite/webaccess"]` on `ep.json`. I even tried using a route like `/api/comment` to avoid authorization, but that was not the reason I was getting 405. And here I got stuck.

Should I create a separate plugin without `"pre": ["ep_etherpad-lite/webaccess"]`? Or is there a better way to handle that?
